plugins/modules/rds_instance.py

```diff
@@ -280,6 +280,14 @@
         description:
           - Specifies if the DB instance is a Multi-AZ deployment. Mutually exclusive with O(availability_zone).
         type: bool
+    multi_tenant:
+        description:
+          - Specifies whether to use the multi-tenant configuration or the single-tenant configuration (default).
+          - This parameter only applies to RDS for Oracle container database (CDB) engines.
+          - The DB engine that you specify in the request must support the multi-tenant configuration.
+          - If the multi-tenant configuration is enabled during creation of the DB instance, it cannot be modified later.
```

Comment:
The `get_options_with_changing_values()` should raise an error if the user is trying to update the `MultiTenant` value from `true` to `false`